cypress/e2e/14-mapping.cy.ts

```diff
@@ -205,63 +205,63 @@ describe('Data mapping', () => {
 				'have.text',
 				`{{ $node['${SCHEDULE_TRIGGER_NODE_NAME}'].json.input[0].count }} {{ $node['${SCHEDULE_TRIGGER_NODE_NAME}'].json.input }}`,
 			);
-		ndv.getters.parameterExpressionPreview('value').should('have.text', ' ');
+		ndv.actions.validateExpressionPreview('value', ' ');
 
 		ndv.actions.selectInputNode('Set');
 
 		ndv.actions.executePrevious();
 		ndv.getters.executingLoader().should('not.exist');
 		ndv.getters.inputDataContainer().should('exist');
-		ndv.getters.parameterExpressionPreview('value').should('include.text', '0 [object Object]');
+		ndv.actions.validateExpressionPreview('value', '0 [object Object]');
 
 		ndv.getters.inputTbodyCell(2, 0).realHover();
-		ndv.getters.parameterExpressionPreview('value').should('include.text', '1 [object Object]');
+		ndv.actions.validateExpressionPreview('value', '1 [object Object]');
 	});
 
-	it('maps keys to path', () => {
-		workflowPage.actions.addInitialNodeToCanvas(MANUAL_TRIGGER_NODE_NAME);
-		workflowPage.getters.canvasNodeByName(MANUAL_TRIGGER_NODE_DISPLAY_NAME).click();
-		workflowPage.actions.openNode(MANUAL_TRIGGER_NODE_DISPLAY_NAME);
-		ndv.actions.setPinnedData([
-			{
-				input: [
-					{
-						'hello.world': {
-							'my count': 0,
-						},
-					},
-				],
-			},
-			{
-				input: [
-					{
-						'hello.world': {
-							'my count': 1,
-						},
-					},
-				],
-			},
-		]);
-
-		ndv.actions.close();
-
-		workflowPage.actions.addNodeToCanvas('Item Lists');
-		workflowPage.actions.openNode('Item Lists');
-
-		ndv.getters.parameterInput('operation').click().find('li').contains('Sort').click();
-
-		ndv.getters.nodeParameters().find('button').contains('Add Field To Sort By').click();
-
-		ndv.getters.inputDataContainer().find('span').contains('my count').realMouseDown();
-
-		ndv.actions.mapToParameter('fieldName');
-
-		ndv.getters.inlineExpressionEditorInput().should('have.length', 0);
-		ndv.getters
-			.parameterInput('fieldName')
-			.find('input')
-			.should('have.value', "input[0]['hello.world']['my count']");
-	});
+	// it('maps keys to path', () => {
```

Comment:
Do we still need this spec? If so, it might be worth to add a comment explaining why it's commented out

Reply:
mistake